src/System.Diagnostics.Process/tests/ProcessTests.OSX.cs

```diff
@@ -16,25 +16,26 @@ namespace System.Diagnostics.Tests
 {
     public partial class ProcessTests : ProcessTestBase
     {
-        [Fact]
-        [OuterLoop("Launches default application")]
-        public void TestWithFilename_ShouldUseOpenWithDefaultApp()
+        [Theory, InlineData("/usr/bin/open"), InlineData("/usr/bin/nano")]
```

Comment:
I agree that this test case is not exercising the usecase where UseShellExecute is true, but I thought it would also be good to show what the expectation of code is when we use Process.Start this way.

Please let me know if there is any objections to the expected behavior or instead this is fine as it.